pkg/operator/sync.go

```diff
@@ -126,6 +144,69 @@ func (optr *Operator) syncTerminationHandler(config *OperatorConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncWebhookConfiguration(config *OperatorConfig) error {
+	err := optr.syncValidatingWebhook(optr.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations(), mapiv1.WithValidatingHooks(config.TargetNamespace))
+	if err != nil {
+		return err
+	}
+	err = optr.syncMutatingWebhook(optr.kubeClient.AdmissionregistrationV1().MutatingWebhookConfigurations(), mapiv1.WithMutatingHooks(config.TargetNamespace))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (optr *Operator) syncValidatingWebhook(client admissionclientv1.ValidatingWebhookConfigurationInterface, hooks []admissionregistrationv1.ValidatingWebhook) error {
+	hook := mapiv1.NewValidatingWebhookConfiguration()
+	hook.Webhooks = hooks
+	validHook, err := optr.validatingWebhookLister.Get(hook.Name)
+	if apierrors.IsNotFound(err) {
+		_, err = client.Create(context.Background(), hook, metav1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		return nil
+	} else if err != nil {
+		return err
+	}
+	if !equality.Semantic.DeepDerivative(hook.Webhooks, validHook.Webhooks) {
+		updateHook := validHook.DeepCopy()
+		updateHook.Annotations = hook.Annotations
+		updateHook.Webhooks = hook.Webhooks
+		_, err = client.Update(context.Background(), updateHook, metav1.UpdateOptions{})
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (optr *Operator) syncMutatingWebhook(client admissionclientv1.MutatingWebhookConfigurationInterface, hooks []admissionregistrationv1.MutatingWebhook) error {
+	hook := mapiv1.NewMutatingWebhookConfiguration()
```

Comment:
Same comments than in https://github.com/openshift/machine-api-operator/pull/626/files#r449617498